### DISCUSSION TITLE
How do you hide certain columns from a model from being shown on the index.html.erb table?

### DISCUSSION THREAD

**Discussion by spacerobotTR** [excid3/madmin#55] [Q&A] [answered]
I have them hidden from the form if I do not need them but I am trying to hide certain fields from the table. How is that done?

**Comment by excid3** [comment, marked as answer]
You add `form: false, index: false, show: false` to the field options to hide from each place. Mix and match as needed.

**Reply by spacerobotTR** [reply]
Thank you!
